### DISCUSSION TITLE
plausible.outbound-links.js not available

### DISCUSSION THREAD

**Discussion by FV-Valraiso** [plausible/analytics#657] [Self-Hosted Support] [answered]
Hi all,
I have a 404, when I try to access plausible.outbound-links.js, on my self hosted instance.
I have no problem for the default plausible.js script.

Thanks for help

**Comment by Vigasaurus** [comment]
What docker tag are you running? And what's the exact route you're trying?

**Reply by metmarkosaric** [reply]
should be working now. was probably related to #655 ?

**Reply by Vigasaurus** [reply]
*Potentially*, but the docker tags aren't automatically updated - Uku updates them manually, I believe. And the most recent Self-Host RC image is at b7a2b3b423a86d6bbbfae92ffc33ac268768c7c8, which is a few days behind the commit that introduced the issue fixed by #655

Unless OP was building a docker image from master manually, that issue shouldn't be affecting it

**Reply by ukutaht** [reply]
`master` and `stable` docker tags are built automatically, versioned releases are managed manually.

**Reply by FV-Valraiso** [reply]
I'm not a docker specialist, I think I'm not using tags.
I used the installation guide : https://plausible.io/docs/self-hosting

I just pulled but didn't get #655

**Reply by FV-Valraiso** [reply]
I think the tag is "latest" ? Found this in the hosting project --> plausible/analytics:latest

**Reply by ukutaht** [reply]
Aha in this case you're using `v1.1` (alias `latest`) which is not updated with outbound-links.js. I will cut a release for `v.1.2` today or tomorrow. Once it's live you can `docker-compose pull` and get access to the new feature.

**Reply by FV-Valraiso** [reply]
Thanks

**Comment by FV-Valraiso** [comment, marked as answer]
V1.2 didn't fix my problem, I still have a 404 on https://[my.self-hosted.domain]/js/plausible.outbound-links.js 

[Edit] I don't see other new features like the dark mode...
I did down, pull and up like said in the release plan :-(

**Reply by ukutaht** [reply]
@FV-Valraiso Ah sorry I update all the tags except `latest`. Can you try pulling again? Should work now.

**Reply by FV-Valraiso** [reply]
Great everything works fine now :-).
Thanks a lot
